modules/seqexec/server/src/main/scala/seqexec/server/nifs/NifsController.scala

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.server.nifs
+
+import cats.Show
+import seqexec.model.dhs.ImageFileId
+import seqexec.server.nifs.NifsController.DCConfig
+import seqexec.server.nifs.NifsController.NifsConfig
+import seqexec.server.ObserveCommand
+import seqexec.server.Progress
+import squants.Time
+import shapeless.tag.@@
+
+trait NifsController[F[_]] {
+
+  def applyConfig(config: NifsConfig): F[Unit]
```

Comment:
This is different from other controllers. Instead of returning `SeqActionF` we just return `F`